Configuration/TSconfig/ContentElementWizard.txt

```diff
@@ -11,3 +11,23 @@ mod.wizards.newContentElement.wizardItems.plugins {
         }
     }
 }
+
+
+#Marker für Configuration/TCA/tx_seminars_seminars.php
+TCEFORM.tx_seminars_seminars.categories.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.event_type.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.place.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.lodgings.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.foods.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.speakers.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.partners.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.tutors.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.leaders.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.organizers.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.organizing_partners.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.target_groups.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.checkboxes.PAGE_TSCONFIG_STR = %
+TCEFORM.tx_seminars_seminars.payment_methods.PAGE_TSCONFIG_STR = %
```

Comment:
Please use a structure like this to avoid duplication:

```
TCEFORM.tx_seminars_seminars.organizers {
  categories.PAGE_TSCONFIG_STR = %
  event_type.PAGE_TSCONFIG_STR = %
  etc.
}
```

Reply:
ok